products/logs/frontend/logsLogic.tsx

```diff
@@ -205,7 +205,7 @@ export const logsLogic = kea<logsLogicType>([
 
                     const response = await api.logs.query({
                         query: {
-                            limit: 999,
+                            limit: 99,
```

Comment:
todo: make this configurable, for now default to smaller number (99 limit here means 100 for some reason, which will be our default once configurable)